tests/peft_integration/test_peft_integration.py

```diff
@@ -815,3 +815,49 @@ def tokenize_function(examples):
                     msg = "When using prompt learning PEFT methods such as PREFIX_TUNING"
                     with self.assertRaisesRegex(RuntimeError, msg):
                         trainer.train()
+
+    def test_peft_pipelines_unnessesary_msg(self):
+        """
+        Test to verify that the warning message "The model 'PeftModel' is not supported for text-generation"
+        does not appear when using PeftModel with text-generation pipeline.
+        """
+        from peft import PeftModel
+
+        from transformers import pipeline
+
+        ADAPTER_PATH = "peft-internal-testing/tiny-OPTForCausalLM-lora"
+        BASE_PATH = "hf-internal-testing/tiny-random-OPTForCausalLM"
+
+        # Input text for testing
+        text = "Who is a Elon Musk?"
+        expected_error_msg = "The model 'PeftModel' is not supported for text-generation"
+
+        model = AutoModelForCausalLM.from_pretrained(
+            BASE_PATH,
+            device_map="auto",
+        )
+        tokenizer = AutoTokenizer.from_pretrained(BASE_PATH)
+
+        lora_model = PeftModel.from_pretrained(
+            model,
+            ADAPTER_PATH,
+            device_map="auto",
+        )
+
+        # Create pipeline with PEFT model while capturing log output
+        pipeline_logger = logging.get_logger("transformers.pipelines.text_generation")
```

Comment:
With the existing logger, the test passes for me, even without your fix!

```suggestion
        pipeline_logger = logging.get_logger("transformers.pipelines.base")
```

Reply:
what is different between  text_generation logger and base logger in transformer library?